### ISSUE TITLE
Could not find a matching normalized version string

### ISSUE THREAD

**Issue by rflaperuta** [CycloneDX/cyclonedx-python#34]
When running `cyclonedx-py -i requirements.txt -o bom.xml`

I got:
```
Input file: requirements.txt
Output BOM: bom.xml
Generating CycloneDX BOM
Traceback (most recent call last):
  File "/home/user/.local/share/virtualenvs/project-mOBEjDrB/bin/cyclonedx-py", line 8, in <module>
    sys.exit(main())
  File "/home/user/.local/share/virtualenvs/project-mOBEjDrB/lib/python3.6/site-packages/cyclonedx/cli/generateBom.py", line 142, in main
    component_elements = read_bom(fd)
  File "/home/user/.local/share/virtualenvs/project-mOBEjDrB/lib/python3.6/site-packages/cyclonedx/cli/generateBom.py", line 103, in read_bom
    raise ValueError("Could not find a matching normalized version string", name, version)
ValueError: ('Could not find a matching normalized version string', 'coverage', '5.0a8')

```

Tracking it down, looks like there is a call to `canonicalize_version` that converts `5.0a8` to `5a8` not giving a match inside `_get_pypi_version` which ultimately raises the exception.

Doing some Q&D editing and removing the call seems too fix it.

Any ideas?

Thx

**Comment by stevespringett** [comment]
Duplicate of #26

**Comment by stevespringett** [comment]
Fixed in v0.3.5
